src/parser.ts

```diff
@@ -6556,7 +6549,7 @@ function parseParenthesizedExpression(
 
   nextToken(parser, context | Context.AllowRegExp | Context.AllowEscapedKeyword);
 
-  const scope = context & Context.OptionsLexical ? addChildScope(createScope(), ScopeKind.ArrowParams) : void 0;
+  const scope = context & Context.OptionsLexical ? new Scope().createChildScope(ScopeKind.ArrowParams) : void 0;
```

Comment:
Not sure why this adds an empty parent scope, change to `new Scope(ScopeKind.ArrowParams)` the tests also passed, should I do it?

Reply:
addVarName, isPrivateIdentifierDefined also uses scope.parent.